.github/workflows/test-move-prover.yml

```diff
@@ -0,0 +1,39 @@
+name: Test_Move_Prover
+
+on:
+  repository_dispatch:
+    types: [new_PR]
+
+jobs:
+  install_and_test_Move_Prover:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Install Z3
+      uses: pavpanchekha/setup-z3@v1.2
+
+    - name: Install .NET Core SDK
+      run: |
+        sudo apt-get update
+        sudo apt-get install apt-transport-https
+        sudo apt-get update
+        sudo apt-get install dotnet-sdk-3.1
+    - name: Install Boogie
+      run: |
+        dotnet tool install --global Boogie
+    - name: Export the environment variable for Boogie
+      run: |
+        echo "::set-env name=BOOGIE_EXE::/home/runner/.dotnet/tools/boogie"
+    - name: Export the environment variable for Z3
+      run: |
+        echo "::set-env name=Z3_EXE::`which z3`"
```

Comment:
can you compress these last 3 steps into a single step?

(nitpick: also some steps have a paragraph break between them (L15 and L35)